recipe/run_conda_forge_build_setup_osx

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
-
 # 2 core available on Travis CI OS X workers: https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
 # CPU_COUNT is passed through conda build: https://github.com/conda/conda-build/pull/1149
 export CPU_COUNT=2
```

Comment:
Do we need to still set this?

Reply:
Yes, on the virtual machines in CI, the default values for these used by conda-build are wrong and since memory is limited, with lots of parallelism, we run out of memory.

We can change the value for azure (while keeping 2 for travis) if you think a different value is better.

Reply:
There's an open issue for azure, https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/47

Reply:
i'd probably do something like
```
export CPU_COUNT="${CPU_COUNT_OVERRIDE:-2}"
```